cmd/ejson/main.go

```diff
@@ -67,7 +72,7 @@ func main() {
 				},
 			},
 			Action: func(c *cli.Context) {
-				if err := decryptAction(c.Args(), c.GlobalString("keydir"), c.String("o")); err != nil {
+				if err := decryptAction(c.Args(), c.GlobalString("keydir"), c.GlobalString("private-key"), c.String("o")); err != nil {
```

Comment:
Should `--private-key` be a global option or local to the decrypt command? I went with global to be consistent with keydir but if it wasn't for that I'd be inclined to make it specific to the decrypt command.